### ISSUE TITLE
Stop process converting if image with color palette - cmyk

### ISSUE THREAD

**Issue by S1mkin** [sindresorhus/gulp-webp#28]
If one image with color palette CMYK, then error and stop converting.

Details:
    code: 255
    killed: false
    stdout: 
    stderr: Unsupported color conversion request
Error! Could not process file /tmp/da030ffb-2c8c-4b43-98d7-d4e26b5f4b99
Error! Cannot read input picture file '/tmp/da030ffb-2c8c-4b43-98d7-d4e26b5f4b99'

How continue after error. plumber() did not help.

**Comment by peter-brennan** [comment]
Did you find a solution. I'm experiencing the same issue, it won't skip the error no matter what I try.


Error in plugin "gulp-webp"
Message:
    Unsupported color conversion request
Error! Could not process file /private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/5154da7a-7fdd-44b9-b17d-014441ea276f
Error! Cannot read input picture file '/private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/5154da7a-7fdd-44b9-b17d-014441ea276f'

Details:
    code: 255
    killed: false
    stdout: 
    stderr: Unsupported color conversion request
Error! Could not process file /private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/5154da7a-7fdd-44b9-b17d-014441ea276f
Error! Cannot read input picture file '/private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/5154da7a-7fdd-44b9-b17d-014441ea276f'

    failed: true
    signal: null
    cmd: /Applications/MAMP/htdocs/website/wp-content/themes/mytheme/node_modules/cwebp-bin/vendor/cwebp -quiet -mt -o /private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/f6125fb8-46f4-4d81-9bd3-e51c5998a370 /private/var/folders/6d/16t37p8x41qc2tml9mj7l7700000gn/T/5154da7a-7fdd-44b9-b17d-014441ea276f
    timedOut: false
    fileName: /Applications/MAMP/htdocs/website/wp-content/uploads/2019/09/Badge-1.jpg

**Comment by S1mkin** [comment]
No, only exclude this files - https://github.com/S1mkin/gulp-img-to-webp
